apps/rehabstod/src/components/Layout/LayoutHeader.tsx

```diff
@@ -26,10 +29,14 @@ export function LayoutHeader() {
                 <IDSIcon height="20" width="20" name="swap" />
                 <Link to="/enhet">Byt vårdenhet</Link>
               </IDSLink>
-              <IDSLink color="var(--IDS-COLOR-PRIMARY-40)" block className="ids-mb-5 text-primary-40">
-                <IDSIcon height="20" width="20" name="cog" />
-                <Link to="/settings">Inställningar</Link>
-              </IDSLink>
+              <IDSDialog dismissible headline="Inställningar" show={showSettingsDialog}>
+                <IDSLink trigger="" color="var(--IDS-COLOR-PRIMARY-40)" block className="ids-mb-5 text-primary-40">
+                  <IDSIcon height="20" width="20" name="cog" />
+                  {/* eslint-disable-next-line jsx-a11y/anchor-is-valid,jsx-a11y/click-events-have-key-events,jsx-a11y/no-static-element-interactions */}
+                  <a onClick={() => setShowSettingsDialog('true')}>Inställningar</a>
+                </IDSLink>
```

Comment:
This should probably be made into a button so that we don't have to disable all the accessibility linting. Take a look at the logout button.

Reply:
I agree but I didn't know how to get the styling but the logout looks perfect. Fixed it now